atomic

```diff
@@ -687,6 +685,8 @@ if __name__ == '__main__':
     try:
         with Atomic.Atomic() as atomic:
             aparser = create_parser(atomic.help())
+            if os.geteuid() != 0:
+                aparser.add_argument(dest="user", action="store_true", default=True)
```

Comment:
It seems a bit odd to create an argument we don't expect people to set - I'd set `self.user` inside `__init__` or so.